front/src/router/index.ts

```diff
@@ -420,10 +420,14 @@ const routes: Array<RouteConfig> = [
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 const scrollBehavior = (to: Route, from: Route, savedPosition: any) => {
   // savedPosition is only available for popstate navigations.
+  const position: { selector?: string; offset?: { x: number; y: number } } = {}
```

Comment:
You define the offset, but you  never set it (it's commented line 435). => Useful ?

Reply:
I remove useless comment.

For vue-router, a position is defined by a selector and an offset. So I prefer to keep the right type.